benchmarks/checkpointing.ts

```diff
@@ -17,25 +17,27 @@ const iterTest = async (numOfIter: number): Promise<Array<number>> => {
     let hrstart = process.hrtime()
     let numOfOps = 0
     let trie = new CheckpointTrie()
-
+    trie.checkpoint()
     for (let i = 0; i < numOfIter; i++) {
       await trie.put(vals[i], keys[i])
-      trie.checkpoint()
       await trie.get(Buffer.from('test'))
       numOfOps++
       if (numOfOps === numOfIter) {
         const hrend = process.hrtime(hrstart)
         resolve(hrend)
       }
     }
+    trie.commit()
```

Comment:
i would think the checkpointing and commit should go *in* the for loop to test the speed of checkpointing many times. the `commit` opens a write stream of the changes/updates back to the main db.

the way it currently is (checkpointing outside of the loop) i think it ends up basically doing the same thing as random.ts